components/kserve/servicemesh_setup.go

```diff
@@ -46,8 +47,9 @@ func (k *Kserve) defineServiceMeshFeatures() feature.FeaturesProvider {
 
 		temporaryFixesErr := feature.CreateFeature("kserve-temporary-fixes").
 			For(handler).
+			ManifestSource(kserveEmbeddedFS).
 			Manifests(
-				path.Join(feature.KServeDir, "grpc-envoyfilter-temp-fix.tmpl"),
+				path.Join(serviceMeshDir, "grpc-envoyfilter-temp-fix.tmpl.yaml"),
```

Comment:
Agree for checking basepath there - [a945320](https://github.com/opendatahub-io/opendatahub-operator/pull/932/commits/a9453203b341c09596f8aafe02e8bf18c8a2c388)

But I think checking for `.patch.` or .`tmp.` would be redundant since `.patch.` contains `.patch`

Reply:
My thinking here is that `.tmp.` is more explicit in terms of format.

Reply:
Ah, I understand now, we can force our naming convention with this. Will do.